### DISCUSSION TITLE
Bagging và Pasting

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#48] [Terminology] [answered]
Chương 7 có thuật ngữ Bagging và Pasting. Đây là hai phương pháp để huấn luyện Cây Quyết định.
* Bagging (viết ngắn gọn từ Bootstrap Aggregating) sử dụng phương pháp lấy mẫu có hoàn lại
* Pasting sử dụng phương pháp lấy mẫu không hoàn lại.

Mình đề xuất không dịch 2 từ ngày. Mọi người nghĩ sao?

**Comment by quangvnai** [comment]
Vậy chắc ta cũng giữ nguyên từ `stacking` giống `ensemble` nhỉ?

**Comment by ngcthuong** [comment]
Lý do không dịch hai từ này là gì nhỉ? Mình nên thử dịch trước xem có được không?

**Reply by minhduc0711** [reply]
có vẻ từ `bootstrap` trong thống kê cũng khá khó dịch ạ

**Comment by quangnhat185** [comment]
Mình đồng ý giữ nguyên. Trong Scikit-Learn đã có sẵn các lớp như `BaggingClassifier` và `BaggingRegression`, nếu giữ nguyên bản gốc thì người đọc sẽ dễ hiểu hơn khi thực hành code.

**Comment by minhduc0711** [comment, marked as answer]
vậy mình giữ nguyên cả 3 từ `bagging`, `pasting` và `stacking` nhé

**Comment by ngcthuong** [comment]
Mình thấy tạm thời giữ nguyên. Đợi khi dịch xong mình quay lại xem xét. Vì nếu để từ mượn nhiều quá cũng không hay lắm.
